aries_cloudagent/config/argparse.py

```diff
@@ -680,7 +680,7 @@ def add_arguments(self, parser: ArgumentParser):
         parser.add_argument(
             "--monitor-revocation-notification",
             action="store_true",
-            env_var="ACAPY_NOTIFY_REVOCATION",
+            env_var="ACAPY_MONITOR_REVOCATION_NOTIF",
```

Comment:
Nit, but I like it when the env variable follows the argument (so I could guess what the env name is from the arg name)

```suggestion
            env_var="ACAPY_MONITOR_REVOCATION_NOTIFICATION",
```